mirrord-layer/tests/http_mirroring.rs

```diff
@@ -214,26 +273,26 @@ async fn test_mirroring_with_http(
     println!("Application subscribed to port, sending tcp messages.");
 
     layer_connection
-        .send_connection_then_data("GET / HTTP/1.1\r\n\r\n")
+        .send_connection_then_data("GET / HTTP/1.1\r\nHost: localhost\r\n\r\n")
```

Comment:
Why was this change necessary?

Reply:
Without it, one of the apps said the request is invalid.